backend/climateconnect_api/utility/email_setup.py

```diff
@@ -295,7 +295,7 @@ def send_email_reminder_for_unread_notifications(
     language_code = get_user_lang_code(user=user)
     subject_by_language = {
         "en": f"You have {total_notifications} unread notifications",
```

Comment:
We also need to change this one to unread messages, otherwise it doesn't fit to the email body

Reply:
You mean change text from`unread notifications` to `unread messages`?